EBSVARS.md

```diff
@@ -2,8 +2,14 @@
 
 As previously mentioned in the [README](README.md), we are using environment variables to make authorized requests to NYPL's API platform. In order to be secure, we are encrypting and decrypting those environment variables using AWS KMS. Please get these variables from someone in the NYPL Digital Department.
 
+ * `KMS_ENV`: Determines whether to interpret ..CLIENT_ID and .._CLIENT_SECRET variables as 'encrypted' or 'unencrypted'. Default 'encrypted'
```

Comment:
Did you mean to italicize everything?

Reply:
I think that formatting issue only happens in the  just in the diff snippet. When viewed as a whole, the github markdown processor seems to implicitly close the `_` https://github.com/NYPL-discovery/discovery-front-end/blob/pb/revise-api-config/EBSVARS.md 

I removed the leading `_` anyway so that no md processor gets similarly confused.